jmclient/setup.py

```diff
@@ -11,6 +11,6 @@
       packages=['jmclient'],
       install_requires=['joinmarketbase==0.9.7dev', 'mnemonic', 'argon2_cffi',
                         'bencoder.pyx>=2.0.0', 'pyaes', 'klein==20.6.0',
-                        'pyjwt==2.1.0', 'autobahn==20.12.3'],
+                        'pyjwt==2.1.0', 'autobahn==20.12.3', 'base58'],
```

Comment:
Maybe pin to a specific version?

Reply:
We definitely don't want to use an external package for that if we can avoid it, which we can I believe?. See:

https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/bbd501962087f682f81db90b332b78c8f1a18f1a/jmbitcoin/jmbitcoin/secp256k1_main.py#L5